pkg/services/baremetal/host/host.go

```diff
@@ -1335,6 +1335,9 @@ func (s *Service) actionDeprovisioning() actionResult {
 			IP:         s.scope.HetznerBareMetalHost.Spec.Status.GetIPAddress(),
 		})
 		out := sshClient.ResetKubeadm()
+		s.scope.Info("ResetKubeadm out.Err", "Err", out.Err)
```

Comment:
you can remove these print statements

Reply:
@janiskemper  ok, I removed all three messages. Now ready to merge?

Reply:
yes